File: strings/keyboard_row.rb
# @param {String[]} words
# @return {String[]}
def find_words(words)
    keyboard = ["qwertyuiop", "asdfghjkl", "zxcvvbnm"]
    res = []

    words.each do |word|
        keyboard.each do |row|
            not_found = false

            word.chars.each do |char|
                unless row.chars.include? char.downcase
                    not_found = true
                    break
                end
            end

            unless not_found
                res << word
                break
            end        
        end
    end
    res
end

File: strings/keyboard_row.py
class Solution:
    def findWords(self, words: List[str]) -> List[str]:
        keyboard = ["qwertyuiop", "asdfghjkl", "zxcvbnm"]
        res = []

        for word in words:
            for row in keyboard:
                not_found = False

                for char in word:
                    if char.lower() not in row:
                        not_found = True
                        break

                if not not_found:
                    res.append(word)
                    break
                
        return res